Sources/Sentry/SentryClient.m

```diff
@@ -634,6 +643,29 @@ - (void)recordLost:(BOOL)eventIsNotATransaction reason:(SentryDiscardReason)reas
     }
 }
 
+- (void)attachScreenshots:(SentryScope *)scope
+{
+#if SENTRY_HAS_UIKIT
+    if (!scope || !self.options.attachScreenshot)
+        return;
+
+    NSArray *shots = [SentryDependencyContainer.sharedInstance.screenshot appScreenshots];
+
+    for (int i = 0; i < shots.count; i++) {
+        NSString *name;
+        if (i == 0)
+            name = @"screenshot.png";
+        else
+            name = [NSString stringWithFormat:@"screenshot-%i.png", i + 1];
```

Comment:
Extend the specs with this name pattern https://develop.sentry.dev/sdk/features/#screenshots

Reply:
I think it should also be possible to send a screenshot with the naming `screenshot-0.png`. This would make the logic simpler here. Otherwise, it would be

- `screenshot.png`
- `screenshot-1.png`
- `screenshot-2.png`

Which is a bit confusing to me. The following is clearer to me:

- `screenshot-0.png`
- `screenshot-1.png`
- `screenshot-2.png`

Reply:
Right now this won't comply with the frontend feature, to show the thumbnail, we require an attachment called `screenshot.png` at least.